vital/types/camera.h

```diff
@@ -40,242 +40,38 @@
 #include <vital/vital_export.h>
 
 #include <iostream>
-#include <vector>
 #include <memory>
 
-#include <vital/vital_config.h>
-#include <vital/types/camera_intrinsics.h>
-#include <vital/types/covariance.h>
-#include <vital/types/rotation.h>
 #include <vital/types/vector.h>
-#include <vital/types/similarity.h>
-#include <vital/logger/logger.h>
 
 
 namespace kwiver {
 namespace vital {
 
-/// forward declaration of camera class
+/// forward declaration of perspective camera class
 class camera;
 /// typedef for a camera shared pointer
 typedef std::shared_ptr< camera > camera_sptr;
 
 
 // ------------------------------------------------------------------
-/// An abstract representation of camera
+/// An abstract representation of perspective camera
 /**
- * The base class of cameras is abstract and provides a
- * double precision interface.  The templated derived class
- * can store values in either single or double precision.
+ * The base class of cameras.
  */
 class VITAL_EXPORT camera
 {
 public:
   /// Destructor
   virtual ~camera() = default;
 
-  /// Create a clone of this camera object
-  virtual camera_sptr clone() const = 0;
```

Comment:
We probably still want to keep `clone` in the base class so we can deep copy cameras without needing to know type